salt/cloud/clouds/azurearm.py

```diff
@@ -52,7 +52,7 @@
     from salt.utils.msazure import object_to_dict
     import azure.storage
     from azure.common.credentials import UserPassCredentials
-    from azure.mgmt.compute import (
+    from azure.mgmt.compute.compute_management_client import (
         ComputeManagementClient,
         ComputeManagementClientConfiguration,
```

Comment:
Configuration classes have disappeared in RC4. Moreover, you don't need to import `compute_management_client`
```python
from azure.mgmt.compute import ComputeManagementClient
```